pkgs/development/python-modules/rasterio/default.nix

```diff
@@ -1,24 +1,25 @@
-{ buildPythonPackage, lib, fetchFromGitHub, isPy3k
+{ buildPythonPackage, lib, fetchFromGitHub, pythonOlder
 , cython, setuptools
-, numpy, affine, attrs, cligj, click-plugins, snuggs, gdal
-, pytest, pytestcov, packaging, hypothesis, boto3, mock
+, matplotlib, numpy, affine, attrs, boto3, cligj, click-plugins, snuggs, gdal
+, pytest, pytestcov, hypothesis, shapely
 }:
 
 buildPythonPackage rec {
   pname = "rasterio";
-  version = "1.1.5";
+  version = "1.2.4";
+  disabled = pythonOlder "3.6";
 
   # Pypi doesn't ship the tests, so we fetch directly from GitHub
   src = fetchFromGitHub {
     owner = "mapbox";
     repo = "rasterio";
     rev = version;
-    sha256 = "168b6hmx026jsvhnq6s5k0qfhzda02mmx1alax6wqk16mk63mqcz";
+    sha256 = "sha256-0tq/0fba+/G9eWq2U1ERqCmEArTLDCiGn30DIJ8oC6g=";
   };
 
-  checkInputs = [ boto3 pytest pytestcov packaging hypothesis ] ++ lib.optional (!isPy3k) mock;
+  checkInputs = [ pytest pytestcov hypothesis shapely ];
```

Comment:
this should be pytestCheckHook.

Reply:
It doesn't work for me with pytestCheckHook (`invalid module name: 'rasterio.errors'`).

Reply:
No idea. Maybe ask on the python nix matrix channel.